config/SENSITIVITY_TEMPERATURES.json

```diff
@@ -0,0 +1,9 @@
+{
+	"version": 1,
+	"application": "loadinsight",
+	"name": "SENSITIVITY_TEMPERATURES",
+	"residential" : {
+		"theat":15,
+		"tcool":25
+	}
```

Comment:
remove the whitespace before the colon after `resedential` and add space after the colon on `theat` and `tcool`

Reply:
fixed